src/main/java/bank/BankMemoryRepository.java

```diff
@@ -0,0 +1,30 @@
+package bank;
+
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.Map;
+
+public class BankMemoryRepository implements BankRepository {
+    private static Map<String, GeneralMember> store = new HashMap<>();
```

Comment:
```suggestion
    private final Map<String, GeneralMember> store = new HashMap<>();
```
하나의 인스턴스로 관리될 repository내에서 굳이 static 선언을 할 필요가 없는 것 같습니다.
static의 쓰임에 대해서 메모리와 관련하여 한번 고민해보면 좋을 것 같습니다.

또한, final키워드를 작성해줌으로써 해당 인스턴스(Map)이 변경되지 않는 다는 것을 명시해야 할 것 같습니다.